src/app/api/course/videoProgress/route.ts

```diff
@@ -26,8 +27,21 @@ export async function GET(req: NextRequest) {
   });
 }
 
+const requestBodySchema = z.object({
+  contentId: z.number(),
+  currentTimestamp: z.number(),
```

Comment:
have u tested this?
does the frontend indeeed send a number or a string?

Reply:
yessir, i have tested